File: scala/Three.scala
//main = do
//       putStrLn "What is the quote?"
//       quote <- getLine
//       putStrLn "Who said it?"
//       author <- getLine
///\       putStrLn $ author ++ " says, \"" ++ quote ++ "\""
//../haskell/Three.hs (END) 

object Three {
  def main(args: Array[String]){  
    println("What is the quote?");
    val quote = readLine();
    println("Who said it?");
    val author = readLine(); 
    println(author+ " says, \"" + quote + "\"")
}
}

File: go/Three.go
package main

import "fmt"

import "os"
import "bufio"
import "strings"

func main() {
	fmt.Println("What is the quote? ")
	var quote string
	//	fmt.Scanln(quote)
	reader := bufio.NewReader(os.Stdin)
	quote, _ = reader.ReadString('\n')
	fmt.Println("Who said it? ")
	who, _ := reader.ReadString('\n')
	fmt.Printf("%s says, \"%s\"", strings.TrimSpace(who), strings.TrimSpace(quote))
}
